.gitignore

```diff
@@ -61,7 +61,6 @@ compile_commands.json
 .ycm_extra_conf.py
 
 # Datafiles
-gphysi.dat
```

Comment:
well, I think it's fine to indeed exclude that one : why did you removed it?

Reply:
At our first PR, we excluded it but It was a mistake. When I noticed it, it was too late and the PR was already merged. According to @sawenzel , it should never be excluded in principle.

Reply:
I think I was fine with putting it into .gitignore. Please remove the change from the PR.